packages/editor/src/panels/files/toolbar.tsx

```diff
@@ -198,6 +198,10 @@ export default function FilesToolbar() {
 
   const { backDirectory, refreshDirectory, createNewFolder } = useCurrentFiles()
 
+  useEffect(() => {
+    refreshDirectory()
+  }, [filesState.refresh.value])
+
```

Comment:
how about we directly call refreshDirectory?
(this way we can only do `when update needed -> refreshDirectory()`
with the state method, we are doing `when update needed -> set refresh value -> listen to refresh value -> refreshDirectory()`)

Reply:
@aditya-mitra 
The places where the refresh is needed are not in reactive contexts.
But the refreshDirectory method requires a reactive context to be called since it involves hooks.